metainfo.patch

```diff
@@ -0,0 +1,50 @@
+diff --git a/miscellaneous/qpdfview.appdata.xml b/miscellaneous/qpdfview.appdata.xml
+index bdcc460..6ddce9a 100644
+--- a/miscellaneous/qpdfview.appdata.xml
++++ b/miscellaneous/qpdfview.appdata.xml
+@@ -32,24 +32,39 @@
+     <launchable type="desktop-id">qpdfview.desktop</launchable>
+     <screenshots>
+       <screenshot type="default">
+-        <image>https://screenshots.debian.net/shrine/screenshot/14328/simage/small-587677349ebef2f9ab1b56d7cbc7888e.png</image>
++        <image>https://i.imgur.com/VzApQWY.png</image>
```

Comment:
I'd just commit them to the PR here under `screenshots/`.

Reply:
Use these links for now https://raw.githubusercontent.com/flathub/flathub/8f5611acd0a2a6cc4a58c130ae8d952f5af9b226/screenshots/1-main.png

You can update it once the submission is accepted into a new repository.